kimchi/js/bindings.js

```diff
@@ -999,8 +999,7 @@ var caml_fp_srs_batch_accumulator_check = function (srs, comms, chals) {
 var caml_fp_srs_batch_accumulator_generate = function (srs, comms, chals) {
     var rust_chals = caml_fp_vector_to_rust(chals);
     var rust_comms = plonk_wasm.caml_fp_srs_batch_accumulator_generate(srs, comms, rust_chals);
-    var rust_comms = caml_array_of_rust_vector(rust_comms, plonk_wasm.WasmGVesta, rust_affine_to_caml_affine, false);
-    return ok;
+    return caml_array_of_rust_vector(rust_comms, plonk_wasm.WasmGVesta, rust_affine_to_caml_affine, false);
```

Comment:
It is weird it worked indeed. Do you know what it was returning?
It seems the only place we use batch_accumulator_generate is in pickles. Should we have tests to be sure it is covered?

Reply:
I'm pretty sure it must have returned `undefined`, so in pickles, if used, it would have hit the `assert false` statement here:
https://github.com/MinaProtocol/mina/blob/4e0b324912017c3ff576704ee397ade3d9bda412/src/lib/pickles/pickles.ml#L1696-L1706

Reply:
To me it seems reasonable that this function could return the actual commitments, but perhaps the reason that this was not required is because the old version just updates some struct inside `plonk_wasm`?

Reply:
Oh, I just saw @mitschabaude's message. Interesting...

Reply:
ooh I understand why it's not called now! the code in `pickles.ml` that calls it is only part of a test (that defines a custom version of `compile`), not part of the actual pickles code

Reply:
Ah! Good to know! Ouf!